.travis.yml

```diff
@@ -6,5 +6,6 @@ directories:
 env:
 language: java
 jdk: oraclejdk8
+dist: trusty
```

Comment:
What was it running with? I.e. what it the default?

Reply:
Yes, and the default was throwing an error saying it wanted a version between 9 and 15. The stated solution was this